src/M2ulPhyS.cpp

```diff
@@ -778,7 +835,9 @@ M2ulPhyS::~M2ulPhyS() {
 
   delete rsolver;
   delete fluxClass;
-  delete mixture;
+  if (transportPtr != NULL) delete transportPtr;
```

Comment:
Is there any point in checking for NULL here?

Reply:
No need of doing it.. Resolved at branch `axis_mixture`.

Reply:
Is there a reason to hold this change on a separate branch?  This is a trivial change, so it doesn't really matter either way I guess, but why not just do it here?

Reply:
Brought the commit from `axis_mixture`.

Reply:
Thanks @dreamer2368